### PR TITLE
Update maintainers

### PR DESCRIPTION
**Pull request by audrow**
Update maintainers to Audrow Nash and Michael Carroll (@mjcarroll).

### CODE REVIEW

File: rmw_implementation/package.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <name>rmw_implementation</name>
   <version>2.6.0</version>
   <description>Proxy implementation of the ROS 2 Middleware Interface.</description>
-  <maintainer email="alejandro@openrobotics.org">Alejandro Hernandez Cordero</maintainer>
```
Comment (clalancette) [inline rmw_implementation/package.xml:7]: Let's make sure to convert old maintainers to authors to recognize their contribution.

(same below)

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rmw_implementation/package.xml (hunk 1 above)
Comment (audrow) [reply rmw_implementation/package.xml:7]: Ah, I didn't know to do this - will do!

Comment (audrow) [reply rmw_implementation/package.xml:7]: Is this true for `setup.py` files as well?

**Comment by audrow** [comment]
@ros-pull-request-builder retest please

**Comment by audrow** [comment]
@ros-pull-request-builder retest this please

**Comment by clalancette** [comment]
This needs a release of `rcpputils` before the Rpr job is going to work properly.  I'll go ahead and do that.

**Review by clalancette — APPROVED** [review]

**Comment by audrow** [comment]
@ros-pull-request-builder retest this please

**Pull request merged**